### DISCUSSION TITLE
SoftBody3D not inside `scene/3d/physics`. Why?

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#10825] [3D]
Hello!

It is rather a question or just observation...

I wandered inside the code and found one strange thing. We have a folder `scene/3d/physics` where we have all the physics nodes. Except one. SoftBody3D is placed just in`scene/3d`. Why? It is strange.

**Comment by AThousandShips** [comment]
It's not a physics class, it's a visual class, it extends `MeshInstance3D`, so it's *more* visual than physical I'd say (unlike `SpringArm3D`)

**Reply by ghost** [reply]
By looking at code it even uses physics server...
